### PR TITLE
fixed broken qplot label

### PR DESCRIPTION
**Pull request by nabaskes**
There are a few other broken things happening in qplot, so I may try and fix them as time goes on.

Thanks for maintaining this project

### CODE REVIEW

File: ggplot/qplot.py [hunk 1]
```diff
@@ -7,13 +7,11 @@
...
 from .scales.scale_log import scale_x_log, scale_y_log
 import pandas as pd
-import numpy as np
 import six
 
-
 def qplot(x, y=None, color=None, size=None, fill=None, data=None,
-        geom="auto", stat=[], position=[], xlim=None, ylim=None, log="",
-        main=None, xlab=None, ylab="", asp=None):
+          geom="auto", stat=[], position=[], xlim=None, ylim=None, log="",
```
Comment (sushinoya) [inline ggplot/qplot.py]: Could you fix the indentation on these two lines?

File: ggplot/qplot.py [hunk 2]
```diff
@@ -69,8 +67,8 @@ def qplot(x, y=None, color=None, size=None, fill=None, data=None,
     >>> print qplot('mpg', data=mtcars, geom="hist", main="hist")
     >>> print qplot('mpg', data=mtcars, geom="histogram", main="histogram")
     >>> print qplot('cyl', 'mpg', data=mtcars, geom="bar", main="bar")
-    >>> print qplot('mpg', 'drat', data=mtcars, xlab= "x lab", main="xlab")
-    >>> print qplot('mpg', 'drat', data=mtcars, ylab = "y lab", main="ylab")
+    >>> print qplot('mpg', 'drat', data=mtcars, xlabel= "x lab", main="xlab")
```
Comment (sushinoya) [inline ggplot/qplot.py]: I know these are just examples and maybe i'm being too naggy here but we should decide if we want spaces before and after the `=`. Its too small an issue so we can leave it as it is. Thanks :)

**Review by sushinoya — COMMENTED** [review]
Hey @nabaskes! Thank you so much for your PR! I've left just two very minor comments. Should be good to merge after that :)

File: ggplot/qplot.py (hunk 1 above)
Comment (nabaskes) [reply ggplot/qplot.py]: I think it's fixed (looks fine in my emacs) although if it's still showing up wrong for you I'll fix in the browser

File: ggplot/qplot.py (hunk 2 above)
Comment (nabaskes) [reply ggplot/qplot.py]: No, PEP8 is important.  This goes to show that I'm over-reliant on my linter.  Should be fixed now.

Thanks!

File: ggplot/qplot.py (hunk 1 above)
Comment (sushinoya) [reply ggplot/qplot.py]: Hey! Yeah it still seems to be off :/

Comment (nabaskes) [reply ggplot/qplot.py]: Okay I see, do you want the new lines consistent with how it was before?

**Pull request merged**